derive/Cargo.toml

```diff
@@ -7,7 +7,8 @@ repository = "https://github.com/ClickHouse/clickhouse-rs"
 homepage = "https://clickhouse.com"
 edition = "2021"
 license = "MIT OR Apache-2.0"
-rust-version = "1.67.0" # update `Cargo.toml` and CI if changed
+# update `Cargo.toml` and CI if changed
+rust-version = "1.70.0"
```

Comment:
It needs to be bumped in the CI as well:
- https://github.com/ClickHouse/clickhouse-rs/blob/main/.github/workflows/ci.yml#L14
- looks like we also don't need [this](https://github.com/ClickHouse/clickhouse-rs/blob/main/.github/workflows/ci.yml#L35-L37), cause Tokio is 1.70 already

Reply:
done

Reply:
Only 1.39+ tokio requires 1.70 rustc, but this crate supports tokio v1.0, so it's invalid point.

What exactly is used here that requires 1.70 rustc?

Reply:
Latest `hyper-rustls` requires 1.70